src/DssSpell.t.sol

```diff
@@ -8,6 +21,22 @@ import "dss-interfaces/Interfaces.sol";
 
 contract DssSpellTest is DssSpellTestBase {
 
+    function testUNIALerpOffboarding() public {
+        checkIlkLerpOffboarding("UNI-A", "UNI-A Offboarding", 165, 1_300);
+    }
+
+    function testUNIV2DAIETHALerpOffboarding() public {
+        checkIlkLerpOffboarding("UNIV2DAIETH-A", "UNIV2DAIETH-A Offboarding", 120, 2_000);
+    }
+
+    function testUNIV2WBTCETHALerpOffboarding() public {
+        checkIlkLerpOffboarding("UNIV2WBTCETH-A", "UNIV2WBTCETH-A Offboarding", 145, 2_400);
+    }
+
+    function testUNIV2WBTCDAIALerpOffboarding() public {
+        checkIlkLerpOffboarding("UNIV2WBTCDAI-A", "UNIV2WBTCDAI-A Offboarding", 120, 800);
+    }
+
```

Comment:
Should also enable `testNewChainlogValues` and test the version bump + the new starknet core address.